### PR TITLE
Finished the add to favorites buttons on all my components

### PR DESCRIPTION
**Pull request by hnavar**
Everything should be working properly

### CODE REVIEW

File: client/src/components/MovieByRatings.tsx [hunk 1]
```diff
@@ -41,10 +53,12 @@ const MovieByRating:FC = () => {
         <Button variant="contained" id="outlined-basic" color="primary" onClick={() => {getMovieData('R')}}>Find NC-17 rated movies</Button>
         <Button variant="contained" id="outlined-basic" color="primary" onClick={() => {getMovieData('NC-17')}}>Find R rated movies</Button>
         <div>
+        <Button type="submit" onClick={() => {saveMovie()}} variant="contained" id="outlined-basic" color="primary">Add movie to favorites</Button>
         <Button variant="contained" id="outlined-basic" color="primary" onClick={() => {setCounter(count + 1)}}>Show Next Movie</Button>
         <Button variant="contained" id="outlined-basic" color="secondary" onClick={() => {setCounter(count - 1)}}>Show Previous Movie</Button>
         </div>
     <div>
+      {console.log(movieData)}
```
Comment (badams92) [inline client/src/components/MovieByRatings.tsx]: remove console logs after testing pls

File: client/src/components/MoviesByPerson.tsx [hunk 2]
```diff
@@ -8,26 +8,36 @@ import {Card, CardHeader, CardMedia, CardContent, Typography} from '@mui/materia
...
   const [searchResults, setSearchResults] = useState<any>([]);
   const [count, setCounter] = useState(0);
 
 
 
-const grabMovies = (actorOrDirector: {name: string}) => {
+  const grabMovies = (actorOrDirector: {name: string}) => {
     return axios.post('/api/movies/moviesByActorOrDirectors/', actorOrDirector)
     .then(({data}: any) => {
       console.log(data);
```
Comment (badams92) [inline client/src/components/MoviesByPerson.tsx]: here too

**Comment by carlbenjaminlyon** [comment]
Outside of needing to remove the console logs, looks good to me.

File: client/src/components/movieSearch.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const SearchMovie:FC = (props :any) => {
     // return axios.get(`https://imdb-api.com/en/API/Title/k_4pd82hff/${searchResults?.imDbId}`)
     return axios.get(`https://imdb-api.com/en/API/Title/${key}/${searchResults?.imDbId}`)
     .then((data: any) => {
-      // console.log(data.data);
+      console.log(data.data);
```
Comment (badams92) [inline client/src/components/movieSearch.tsx]: this one too

File: client/src/components/movieSearch.tsx [hunk 4]
```diff
@@ -53,6 +53,24 @@ const SearchMovie:FC = (props :any) => {
     })
   }
 
+  const saveMovie = () => {
+    addMovieInfo();
+    if(user) {
+      console.log(
+        {movieId: searchResults?.imDbId,
+        userId: user.id}
+        )
```
Comment (badams92) [inline client/src/components/movieSearch.tsx]: and here

**Comment by badams92** [comment]
looks good

**Comment by badams92** [comment]
I like the new changes

File: server/api/movies.ts [hunk 5]
```diff
@@ -1,7 +1,8 @@
 import {Router} from 'express';
 import type {Request, Response} from 'express';
-import { getAllMovies, getAllMoviesByDirector, getAllMoviesByGenre, getAllMoviesWithActor, addMovie, grabMovieIdWithRating, grabMoviesByActorsOrDirectors, grabActorOrDirectorID, getMovieById, getFavoriteMovies } from '../database/index';
+import { getAllMovies, getAllMoviesByDirector, getAllMoviesByGenre, getAllMoviesWithActor, addMovie, grabMovieIdWithRating, grabMoviesByActorsOrDirectors, grabActorOrDirectorID, getMovieById, getFavoriteMovies,  } from '../database/index';
```
Comment (carlbenjaminlyon) [inline server/api/movies.ts:3]: Probably not a problem, just an extra comma at the end of getFavoriteMovies

File: client/src/components/MoviesByPerson.tsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ import {Card, CardHeader, CardMedia, CardContent, Typography} from '@mui/materia
 
 
 
-const MoviesByPerson:FC<any> = ({user}) => {
+const MoviesByPerson:FC = () => {
   const [searchVal, setSearchVal] = useState('');
```
Comment (badams92) [inline client/src/components/MoviesByPerson.tsx:12]: won't we need the user info for saving that movie to the user's movies?

**Pull request merged**